client/js/gdpr/chart/index.js

```diff
@@ -116,35 +127,6 @@ export function createChart(el, rankingHistory, rankingDates) {
   el.append(tooltip);
 }
 
-/**
- * Tests whether the mouse is currently near a point on the chart.
- *
- * @param {SVGSVGElement} svg The SVG element for the chart.
- * @param {number[][]} rows The ranks of each series at each date.
- * @param {number} clientX The position of the mouse.
- * @param {number} clientY The position of the mouse.
- * @returns {{hit: number | null, x: number | null, y:number | null}} A hit
- * result.
- */
-function hitTest(svg, rows, clientX, clientY) {
-  // convert mouse coords to SVG coords
-  const mousePos = svg.createSVGPoint();
-  mousePos.x = clientX;
-  mousePos.y = clientY;
-  const chartPos = mousePos.matrixTransform(svg.getScreenCTM().inverse());
-
-  // index in history
-  const x = Math.round(chartPos.x / RUN_SCALE_X);
-  // value in history
-  const y = Math.round(chartPos.y / RUN_SCALE_Y);
-
-  const hit = rows[x].findIndex((rank) => rank === y);
-
-  if (hit < 0) return {hit: null, x: null, y: null};
-
-  return {hit, x, y};
-}
-
```

Comment:
is this deleted just because most of it is added above in lines 59-68? What's your reason for deleting it? 

Your `svg.addEventListener('mousemove', ({clientX, clientY}) => {` is getting kind of long so I would avoid adding more lines

Reply:
The function felt clunky and prevented me from doing [this optimization](https://github.com/googleinterns/step99-2020/blob/761d86798a14dee9ac2474c85fcd6af9a2a17651/client/js/chart/index.js#L63), so I scrapped it.